app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt

```diff
@@ -52,34 +53,25 @@ class VideoConferenceActivity : JitsiMeetActivity(), JitsiVideoConferenceView,
     override fun onConferenceJoined(map: MutableMap<String, Any>?) =
         logJitsiMeetViewState("Joined video conferencing", map)
 
-    override fun onConferenceWillLeave(map: MutableMap<String, Any>?) =
-        logJitsiMeetViewState("Leaving video conferencing", map)
-
-    override fun onConferenceLeft(map: MutableMap<String, Any>?) {
-        logJitsiMeetViewState("Left video conferencing", map)
-        finishJitsiVideoConference()
+    override fun onConferenceTerminated(map: MutableMap<String, Any>?) {
+        if(map!!.containsKey("error")) {
```

Comment:
Is there a place where I can find all the keys like the `error` for the map?

Reply:
@filipedelimabrito Actually the Jitsi documentation specified the keys for Map. if the `onConferenceTerminated` ended successfully no `error` key will be present else `error` key will be there.

Reply:
Thanks for pointing it out @Cool-fire.